change/office-ui-fabric-react-d4d38137-bba4-4a45-a5f9-d94fb3ed6ea4.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added PivotLinkFormatType and PivotLinkSizeType",
```

Not sure if this is needed for v7, feel free to ignore.

```suggestion
  "comment": "feat: Added PivotLinkFormatType and PivotLinkSizeType.",
```